ravenframework/Optimizers/fitness/fitness.py

```diff
@@ -57,57 +59,57 @@ def invLinear(rlz,**kwargs):
           the farthest from violating the constraint it is, The highest negative value it have the largest the violation is.
     @ Out, fitness, xr.DataArray, the fitness function of the given objective corresponding to a specific chromosome.
   """
-  if kwargs['a'] == None:
-    a = 1.0
-  else:
-    a = kwargs['a']
-  if kwargs['b'] == None:
-    b = 10.0
-  else:
-    b = kwargs['b']
-  if kwargs['constraintFunction'].all() == None:
-    penalty = 0.0
-  else:
-    penalty = kwargs['constraintFunction'].data
-
-  objVar = kwargs['objVar']
-  data = np.atleast_1d(rlz[objVar].data)
-
-  fitness = -a * (rlz[objVar].data).reshape(-1,1) - b * np.sum(np.maximum(0,-penalty),axis=-1).reshape(-1,1)
-  fitness = xr.DataArray(np.squeeze(fitness),
+  a = [1.0]                    if kwargs['a'] == None                         else kwargs['a']
+  b = [10.0]                   if kwargs['b'] == None                         else kwargs['b']
+  penalty = 0.0                if kwargs['constraintFunction'].all() == None  else kwargs['constraintFunction'].data
+  objVar = [kwargs['objVar']]  if isinstance(kwargs['objVar'], str) == True   else kwargs['objVar']
+
+  for j in range(len(objVar)):
+    data = np.atleast_1d(rlz[objVar][objVar[j]].data)
+    fitness = -a[0] * (rlz[objVar][objVar[j]].data).reshape(-1,1) - b[0] * np.sum(np.maximum(0,-penalty),axis=-1).reshape(-1,1)
+    fitness = xr.DataArray(np.squeeze(fitness),
```

Comment:
What does hardcoding zero mean? what if a list has different values? I feel this can be better.

Reply:
"hardcoding zero" here is just for extracting single element value from a list. a and b are arguments used for fitness calculation. Since the type of kwargs['a']  is a list, I believe this is the best way to extract value from the list. Any suggestions...?